test/point_test.cpp

```diff
@@ -34,16 +34,16 @@ TEST_CASE("Point tests", "[Point]")
     {
         auto point0 = Point(0);
         // Position
-        point0.UnscaledX(INT32_MAX);
-        point0.UnscaledY(INT32_MAX);
-        point0.UnscaledZ(INT32_MAX);
-        REQUIRE(point0.UnscaledX() == INT32_MAX);
-        REQUIRE(point0.UnscaledY() == INT32_MAX);
-        REQUIRE(point0.UnscaledZ() == INT32_MAX);
+        point0.UnscaledX(std::numeric_limits<int>::max());
```

Comment:
I think you can replace these with their const-width versions:

`std::numeric_limits<int32_t>::max()`